src/main/java/pl/edu/mimuw/chess/Player.java

```diff
@@ -0,0 +1,124 @@
+package pl.edu.mimuw.chess;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.HashSet;
+
+import static pl.edu.mimuw.chess.V2.*;
+import static pl.edu.mimuw.chess.ChessBoard.*;
+
+public class Player {
+	private final ChessColor color;
+	private Set<AbstractPiece> pieces;
+	
+	public Player(ChessColor color) {
+		this.color = color;
+		
+		this.pieces = new HashSet<AbstractPiece>();
```

Comment:
to nie jest poprawny kod w ogólności, bo nie widze zeby pieces mialo dobre equals/hashcode (tu zadzialaly Ci dobrze te domysle implementacje tych metod, ale tak nie powinno sie robic)